src/main/io/osd.c

```diff
@@ -521,22 +533,15 @@ void osdInit(displayPort_t *osdDisplayPortToUse)
 
     displayClearScreen(osdDisplayPort);
 
-    // display logo and help
-    char x = 160;
-    for (int i = 1; i < 5; i++) {
-        for (int j = 3; j < 27; j++) {
-            if (x != 255)
-                displayWriteChar(osdDisplayPort, j, i, x++);
-        }
-    }
```

Comment:
Yeah, I was wondering wether or not to update the `display` api to provide a 'displayDrawBlock' or similar method, at this time I decided not to and to keep the logo code in the `osd` api while I discover more about the new code.